### DISCUSSION TITLE
Github Student

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#33] [Q&A] [answered]
Have you registered for https://education.github.com/?

**Comment by TechStudent10** [comment, marked as answer]
No? Why are you asking me these questions?

**Reply by KendallDoesCoding** [reply]
Ok, i thought it looks interesting and also you get the pro badge and other cool stuff so was wondering if you registered... Sorry if it comes across at weird though, Ik your feeling a little weird with questions like this and #27 ... Sorry!!

**Comment by KendallDoesCoding** [comment]
I got accepted into the program today, and got this epic badge and amazing offers, free stuff!

![image](https://user-images.githubusercontent.com/84712013/163126212-2166737c-8ec6-4667-85e6-bccaa0b7309c.png)

**Reply by KendallDoesCoding** [reply]
@TechStudent11 If your wondering, here's all the benefits you get: https://education.github.com/pack

You get 2-3 domains (the third one applies only if your school is a partner of namecheap, but if not you can get a free .tech, bunch of other domain names for 1 year), you can get a SSL certificate, some courses, the github pro badge with it's benefits, etc.

**Reply by TechStudent10** [reply]
You can get GitHub Pro for free? Anyways, of the benefits, the Canva, Replit, GitHub and Dashlane ones do look awesome. Still probably won't be applying though.

I am confused, however, as to how you got the `Developer Program Member` badge. From what I can see, none of your projects integrate  with GitHub or it's API.

**Reply by KendallDoesCoding** [reply]
> You can get GitHub Pro for free? 

Yeah, that's a benifit of applying for the student plan and getting accepted

> Anyways, of the benefits, the Canva, Replit, GitHub and Dashlane ones do look awesome.
Yeah exactly, even the free domain ones like .tech, .me, .live, .software, .games, etc look great to me

> Still probably won't be applying though.
I mean fair enough, you may not want to approach your school, so you can see all the ways you can apply without approaching the school if your still interested, like a ss of the school portal, etc.

> I am confused, however, as to how you got the Developer Program Member badge. From what I can see, none of your projects integrate with GitHub or it's API.

Signed up for that here: https://github.com/settings/profile#github-developer-program, you can join the dev program and get the badge.

**Reply by TechStudent10** [reply]
> Signed up for that here: https://github.com/settings/profile#github-developer-program, you can join the dev program and get the badge.

From what I can see on that page:

> Building an application, service, or tool that integrates with GitHub? [Join the GitHub Developer Program](https://github.com/developer/register), or read more about it at our [Developer site](https://docs.github.com/).

So I'm really confused. What projects have you made that use the GitHub API? Otherwise, that badge is invalid.

**Reply by KendallDoesCoding** [reply]
> > Signed up for that here: https://github.com/settings/profile#github-developer-program, you can join the dev program and get the badge.
> 
> From what I can see on that page:
> 
> > Building an application, service, or tool that integrates with GitHub? [Join the GitHub Developer Program](https://github.com/developer/register), or read more about it at our [Developer site](https://docs.github.com/).
> 
> So I'm really confused. What projects have you made that use the GitHub API? Otherwise, that badge is invalid.

Dude, to be honest, idk how it works, I just registered and the perks are apparently: 

Notification of API changes
Early access eligibility on select features
Eligibility for development licenses for GitHub products
GitHub profile membership badge

You can register too https://github.com/developer/register

**Reply by KendallDoesCoding** [reply]
The dev badge also shows on most of my orgs which I've registered for it on, again idrk how the badge works, I just registered for it coz I was feeling bored.

**Reply by TechStudent10** [reply]
Ok makes sense.
